config/config.go

```diff
@@ -36,6 +38,9 @@ func Get() *Config {
 	if v := os.Getenv("CODELIST_API_URL"); len(v) > 0 {
 		cfg.CodeListAPIURL = v
 	}
+	if v := os.Getenv("DATASET_API_URL"); len(v) > 0 {
+		cfg.CodeListAPIURL = v
```

Comment:
this should be cfg.DatasetAPIURL? - might be a merge conflict as there was an issue with one of these urls being misassigned during the deployment work